sql/firefox_accounts_derived/fxa_log_content_other_events_v1/query.sql

```diff
@@ -0,0 +1,71 @@
+SELECT
+  jsonPayload.event,
+  jsonPayload.flow_time,
+  jsonPayload.locale,
+  jsonPayload.useragent,
+  jsonPayload.country,
+  jsonPayload.entrypoint,
+  jsonPayload.flow_id,
+  jsonPayload.region,
+  jsonPayload.service,
+  jsonPayload.utm_campaign,
+  jsonPayload.utm_content,
+  jsonPayload.utm_medium,
+  jsonPayload.utm_source,
+  -- timestamp, -- This is a static value in 1970 for all records
+  receiveTimestamp
+FROM
+  `moz-fx-fxa-prod-0712.fxa_prod_logs.docker_fxa_content_19700101`
+WHERE
+  NOT (
+    REGEXP_CONTAINS(jsonPayload.event, r"^flow\.([\w-]+)\.submit$")
+    OR REGEXP_CONTAINS(
+      jsonPayload.event,
+      r"^experiment\.(?:control|designF|designG)\.passwordStrength\.([\w]+)$"
+    )
+    OR REGEXP_CONTAINS(jsonPayload.event, r"^flow\.signin-totp-code\.submit$")
+    OR REGEXP_CONTAINS(jsonPayload.event, r"^screen\.signin-totp-code$")
+    OR REGEXP_CONTAINS(jsonPayload.event, r"^flow\.signin-totp-code\.engage$")
+    OR REGEXP_CONTAINS(jsonPayload.event, r"^screen\.settings\.two-step-authentication$")
+    OR REGEXP_CONTAINS(jsonPayload.event, r"^flow\.([\w-]+)\.engage$")
+    OR REGEXP_CONTAINS(jsonPayload.event, r"^flow\.[\w-]+\.forgot-password$")
+    OR REGEXP_CONTAINS(jsonPayload.event, r"^flow\.[\w-]+\.have-account$")
+    OR REGEXP_CONTAINS(jsonPayload.event, r"^flow\.((?:install|signin)_from)\.\w+$")
+    OR REGEXP_CONTAINS(
+      jsonPayload.event,
+      r"^flow\.connect-another-device\.link\.(app-store)\.([\w-]+)$"
+    )
+    OR REGEXP_CONTAINS(jsonPayload.event, r"^screen\.(?:oauth\.)?([\w-]+)$")
+    OR REGEXP_CONTAINS(
+      jsonPayload.event,
+      r"^settings\.communication-preferences\.(optIn|optOut)\.success$"
+    )
+    OR REGEXP_CONTAINS(jsonPayload.event, r"^settings\.clients\.disconnect\.submit\.([a-z]+)$")
+    OR REGEXP_CONTAINS(jsonPayload.event, r"^([\w-]+).verification.clicked$")
+    OR REGEXP_CONTAINS(jsonPayload.event, r"^flow\.signin-totp-code\.success$")
+    OR REGEXP_CONTAINS(jsonPayload.event, r"^flow\.(support)\.success$")
+    OR REGEXP_CONTAINS(jsonPayload.event, r"^flow\.(support)\.view$")
+    OR REGEXP_CONTAINS(jsonPayload.event, r"^flow\.(support)\.fail$")
+    OR REGEXP_CONTAINS(jsonPayload.event, r"^flow\.([\w-]+)\.view$")
+  )
+  AND jsonPayload.event NOT IN (
+    'flow.reset-password.submit', --regex
```

Comment:
What are these `-- regex` comments about?

Reply:
Probably should have removed them, sorry about that. I just noted which events explicitly mapped are also caught by regexp in the same query. So having them caught twice isn't necessary, but I left them anyways because that's how it's laid out in the code with the mappings.

Reply:
I think the comments are helpful, then. I will plan to keep or expand them.

Reply:
> that's how it's laid out in the code with the mappings

Can you point me to the code you're referencing? It would be good to link that in a comment in the query text here.